<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Войдите в библиотеку</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
	
	background-image: url(:/img/ok.jpg);
	
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>850</width>
    <height>800</height>
   </size>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>591</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
font-family: century gothic;
font-size:14px;
border-radius:60px;
background-color:#112e42;
border-color:#112e42;
}




</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
font-family: century gothic;
font-size:14px;
color:white;
}</string>
     </property>
     <property name="text">
      <string>Войдите в систему здесь</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enter_login">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>430</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#enter_login{
background:#00aeff;
border-radius:15px;
color:white;
}
#enter_login:hover{
	color:whitt ;
	background: #d58a13;
}</string>
     </property>
     <property name="text">
      <string>Входить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancel_login">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>430</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#cancel_login{
background:#00aeff;
border-radius:15px;
color:white;
}
#cancel_login:hover{
	color:whitt ;
	background: #d58a13;
}</string>
     </property>
     <property name="text">
      <string>Выход</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="username">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#username{
	background: transparent;
	border:none;
	color: #717072;
	border-bottom: 1px solid #717072
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#password{
	background: transparent;
	border:none;
	color: #717072;
	border-bottom: 1px solid #717072
}
</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="_checkBox_Home">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#checkBox_Home{
	color:white;
}</string>
     </property>
     <property name="text">
      <string>Agree with terms</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Register_Home">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>320</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#Register_Home{
	background:black;
	border-radius:15px;
	background-color: rgb(70, 85, 127);
	color:white;
}
#Register_Home:hover{
	border-radius:15px;
	background-color: rgb(0, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>зарегистрировать</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ho">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>270</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_ho{
	color: white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="icon2_home">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#icon2_home{
	background-image: url(:/img/login-icon.png);
	border-radius:25px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
